train.py

```diff
@@ -107,18 +107,31 @@
         code_col=1,
         description_col=0,
         authoritative=True,
+        multiplier = 5,
         creates_codes=False,
     )
 )
 
-data_sources.append(SearchReferencesDataSource())
+data_sources.append(
+    BasicCSVDataSource(
+        args.Brands_data_file(),
+        code_col=1,
+        description_col=0,
+        authoritative=True,
+        multiplier = 3,
+        creates_codes=False,
+    )
+)
+
+data_sources.append(SearchReferencesDataSource()) ##Can we add this twice?
```

Comment:
We can definitely do this. We can actually incorporate this using the multiplier